platform/native/contiki-main.c

```diff
@@ -242,6 +242,8 @@ main(int argc, char **argv)
 
   serial_line_init();
 
+  process_start(&sensors_process, NULL);
```

Comment:
I understand what this does. I just don't think everybody wants to use this sensors. So instead of assuming the sensors are always on, applications that require sensors should start them manually.